lib/algos/list/delimiterOrphanDeleteMarker.js

```diff
@@ -1,45 +1,35 @@
-'use strict'; // eslint-disable-line strict
-const Delimiter = require('./delimiter').Delimiter;
-const VSConst = require('../../versioning/constants').VersioningConstants;
-const { inc, FILTER_ACCEPT, FILTER_END, SKIP_NONE } = require('./tools');
-const VID_SEP = VSConst.VersionId.Separator;
-const { DbPrefixes } = VSConst;
-
+const DelimiterVersions = require('./delimiterVersions').DelimiterVersions;
+const { FILTER_END } = require('./tools');
 const DELIMITER_TIMEOUT_MS = 10 * 1000; // 10s
 const TRIM_METADATA_MIN_BLOB_SIZE = 10000;
-
 /**
- * Handle object listing with parameters. This extends the base class Delimiter
+ * Handle object listing with parameters. This extends the base class DelimiterVersions
  * to return the orphan delete markers. Orphan delete markers are also
  * refered as expired object delete marker.
  * They are delete marker with zero noncurrent versions.
  */
-class DelimiterOrphanDeleteMarker extends Delimiter {
+class DelimiterOrphanDeleteMarker extends DelimiterVersions {
     /**
-     * Delimiter listing of non-current versions.
+     * Delimiter listing of orphan delete markers.
      * @param {Object}  parameters            - listing parameters
      * @param {String}  parameters.beforeDate - limit the response to keys older than beforeDate
      * @param {RequestLogger} logger          - The logger of the request
      * @param {String} [vFormat]              - versioning key format
      */
     constructor(parameters, logger, vFormat) {
         super(parameters, logger, vFormat);
-
+        this.keyMarker = parameters.marker;
```

Comment:
Is `this.keyMarker` used anywhere?

Reply:
`this.keyMarker` is used by `DelimiterVersions`.

Reply:
The orphan delete marker logic uses the term 'marker' instead of 'keyMarker', as the latter could suggest the presence of a 'versionIdMarker'.

Reply:
I can still update the constructor like so: 
```
constructor(parameters, logger, vFormat) {
        parameters.keyMarker = parameters.marker;
        super(parameters, logger, vFormat);
```

Reply:
Or just pass a new object instead of `parameters` constructed with the params you need for `DelimiterVersions`, as if you were instanciating it directly.